gateway/crates/federated-server/src/server/graph_fetch_method.rs

```diff
@@ -37,12 +36,12 @@ impl GraphFetchMethod {
         sender: watch::Sender<Option<Arc<Engine>>>,
     ) -> crate::Result<()> {
         match self {
-            #[cfg(not(feature = "lambda"))]
             GraphFetchMethod::FromApi {
                 access_token,
                 graph_name,
                 branch,
             } => {
+                #[cfg(not(feature = "lambda"))]
```

Comment:
?

Reply:
The code is still behind the `not(lambda)` flag